canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -1207,7 +1207,7 @@ export default class PropertiesController {
 		return returnValues;
 	}
 
-	setPropertyValues(values, isInitProps) {
+	setPropertyValues(values, setDefaultValues, isInitProps) {
```

Comment:
Should we change this method to pass in `options` with  `setDefaultValues` and `isInitProps` be props in that object?